uni/lib/view/login/login.dart

```diff
@@ -125,51 +203,52 @@ class LoginPageViewState extends State<LoginPageView> {
       ),
       child: Builder(
         builder: (context) => Scaffold(
+          resizeToAvoidBottomInset: false,
           backgroundColor: darkRed,
           body: BackButtonExitWrapper(
             child: Padding(
               padding: EdgeInsets.only(
                 left: queryData.size.width / 8,
                 right: queryData.size.width / 8,
               ),
-              child: ListView(
+              child: Column(
                 children: [
-                  Padding(
-                    padding: EdgeInsets.only(
-                      bottom: queryData.size.height / 20,
+                  SizedBox(height: queryData.size.height / 20),
+                  SizedBox(
+                    width: 100,
+                    // TODO(thePeras): Divide into two svgs to add color
```

Comment:
Is this TODO going to be fixed in this PR as well?

Reply:
for redesign